module/batch-action/src/Application/Controller/Api/GetTemplatesByBatchAction.php

```diff
@@ -85,7 +85,14 @@ public function __invoke(Request $request): Response
     {
         try {
             /** @var BatchActionFilterFormModel $data */
-            $data = $this->serializer->deserialize($request->getContent() ?? '', BatchActionFilterFormModel::class);
+            $data = $this->serializer->denormalize(
+                $request->query->get('filter') ?? [],
+                BatchActionFilterFormModel::class
+            );
+            $filter = $request->get('filter');
+            if (isset($filter['ids']['included'])) {
```

Comment:
I'd add check for it... Now we won't notify if someone gives it as i.e.  1 and treat it as false... only 'true' and 'false' are appropriate values... Don't we have sth like boolean demonormalizer? I think it could be move to BatchActionFilterIdsFormModelHandler anyway. In the future such things should be handled via serializer logic.